po/zh_CN.po

```diff
@@ -2139,14 +2147,14 @@ msgstr ""
 #: fread.c:1657
 #, c-format
 msgid "(%llu bytes from row 1 to eof) / (2 * %llu jump0size) == %llu\n"
-msgstr ""
+msgstr "(从首行到结束共 %llu bytes) / (2 * %llu jump0size) == %llu\n"
```

Comment:
The full context for these is:

```
  if (verbose) {
    DTPRINT(_("  Number of sampling jump points = %d because "), nJumps);
    if (nrowLimit<INT64_MAX) DTPRINT(_("nrow limit (%llu) supplied\n"), (llu)nrowLimit);
    else if (jump0size==0) DTPRINT(_("jump0size==0\n"));
    else DTPRINT(_("(%llu bytes from row 1 to eof) / (2 * %llu jump0size) == %llu\n"),
                 (llu)sz, (llu)jump0size, (llu)(sz/(2*jump0size)));
  }
```

`jump0size==0` should be fine to leave un-translated.

`nrow limit (%llu) supplied` should come when passing `nrow=` argument to `fread`, which `limit`s the number of rows to read.